### ISSUE TITLE
Crashes in 2.80

### ISSUE THREAD

**Issue by Caleb9** [nutti/Screencast-Keys#11]
**System Information**
  
* OS: Windows 10
* Blender version: 2.80
* Add-on version: master (8070928e15d42384cc764f7cd4fb6e4882582721)

**Expected behavior**  
Plugin works

**Description about the bug**  
I managed to install the plugin but when I enable it, it shows as collapsed menu in the view port. Clicking checkbox in that menu (see the screenshot) results with this error.
```
Traceback (most recent call last):
  File "C:\Users\Piotr\AppData\Roaming\Blender Foundation\Blender\2.80\scripts\addons\Screencast-Keys\ops.py", line 807, in draw
    prefs = compat.get_user_preferences(bpy.context).addons["screencastkeys"].preferences
KeyError: 'bpy_prop_collection[key]: key "screencastkeys" not found'

location: <unknown location>:-1

```

**Screenshots/Files**
![Screenshot (2)](https://user-images.githubusercontent.com/7767011/68998195-78f76680-08af-11ea-8f43-a1aa05935b01.png)

**Comment by Hawkpath** [comment]
It seems that this error is the result of the addon being manually installed on your system. Installing through the preferences menu (in addition to copying the source files to appdata) modifies `...\Blender\2.80\config\userpref.blend` with addon info, and by skipping this, Blender doesn't create any data structures internally which the addon is trying to access here. If we can resolve #12 and get the addon properly installed, then this issue will also be resolved.

**Comment by nutti** [comment]
@Caleb9 

Install from .zip file is not supported in case of master branch.
If you want to install from .zip file, please use .zip file in [Release Page](https://github.com/nutti/Screencast-Keys/releases).
Maybe we should release v3.1 as soon as possible.

If you try to install the add-on in master branch, you must put on add-on folder manually.

**Comment by Caleb9** [comment]
@nutti Is the released version working with Blender 2.81? Looking through commits I think I noticed that support for 2.81 was added after latest release. Otherwise yeah, I tried to put the add-on folder manually, but as @Hawkpath noted, there's some extra work to do then, that I don't exactly know how to go about.

**Comment by nutti** [comment]
@Caleb9 

Sorry for delayed reply.
As I mentioned [here](https://github.com/nutti/Screencast-Keys/issues/12#issuecomment-605729077), installation method differs between unstable version and release version.
After I finished to solve some issues, I will release v3.1.

And, this issue comes from the installation method, so I will close this for now.
If you have any further issue about this, please reopen or create new issue.
